__tests__/AccountSettings.js

```diff
@@ -0,0 +1,67 @@
+import {
+  AWSInfoResponse,
+  getPersistedMockCall,
+  userResponse,
+} from 'testUtils/mockHttpCalls';
+import { renderRouteWithStore } from 'testUtils/renderUtils';
+
+const elementIDs = {
+  ChangeEmailForm: 'account-settings/change-email',
+  ChangePasswordForm: 'account-settings/change-password',
+};
+
+const elementLabels = {
+  Email: 'Email',
+  EmailExplanatoryText:
+    'This address is used for logging in and for all communication. Be aware that it is also visible to other members of your organization.',
+  DeleteAccount: 'Delete Account',
+};
+
+const requests = {};
+
+describe('AccountSettings', () => {
+  beforeAll(() => {
+    requests.user = getPersistedMockCall('/v4/user/', userResponse);
+    requests.info = getPersistedMockCall('/v4/info/', AWSInfoResponse);
+    requests.orgs = getPersistedMockCall('/v4/organizations/');
+    requests.clusters = getPersistedMockCall('/v4/clusters/');
+    requests.catalogs = getPersistedMockCall('/v4/appcatalogs/');
+  });
+
+  afterAll(() => {
+    for (const req of Object.values(requests)) {
+      req.persist(false);
+    }
+  });
```

Comment:
We should probably get rid of this and use the pattern @ubergesundheit  used for organizations from now on.